src/main/scala/io/iohk/ethereum/ledger/Ledger.scala

```diff
@@ -258,7 +281,7 @@ object Ledger extends Logger {
     * @param worldStateProxy
     * @return
     */
-  private def payForGasUsedToBeneficiary(stx: SignedTransaction, worldStateProxy: InMemoryWorldStateProxy): InMemoryWorldStateProxy = worldStateProxy //TODO
+  private def payForGasUsedToBeneficiary(stx: SignedTransaction)(worldStateProxy: InMemoryWorldStateProxy): InMemoryWorldStateProxy = worldStateProxy //TODO
```

Comment:
Isn't this function implemented with the `payMinerForGasFn` function in `executeTransaction`?